ThirdPartyNotices.txt

```diff
@@ -5,6 +5,7 @@ Visual Studio App Center SDK for Android incorporates third party material from
 
 1.	Gradle (http://gradle.org)
 2.	Android Robot image (https://developer.android.com/distribute/tools/promote/brand.html)
+3.  Android Lifecycle Process (https://mvnrepository.com/artifact/androidx.lifecycle/lifecycle-process)
```

Comment:
```suggestion
3.    Android Lifecycle Process (https://mvnrepository.com/artifact/androidx.lifecycle/lifecycle-process)
```

Reply:
Good catch, but let's keep tabs instead of spaces here
```suggestion
3.	Android Lifecycle Process (https://mvnrepository.com/artifact/androidx.lifecycle/lifecycle-process)
```